preheaders/checkout.php

```diff
@@ -819,6 +815,9 @@
 	}
 }
 
+// Preventing conflicts with old checkout templates that depend on the $pmpro_level global being set.
+pmpro_getAllLevels();

```

Comment:
Curious how to test just this one piece. Why we need this and how we'd know it is or isn't needed.

Reply:
Confirming this is needed and resolving.